client/flutter/lib/components/page_button.dart

```diff
@@ -47,7 +47,8 @@ class PageButton extends StatelessWidget {
             children: <Widget>[
               Text(
                 this.title,
-                textScaleFactor: 1.0 + 1.0 * contentScale(context),
+                // textScaleFactorOf is for the device font size
+                textScaleFactor: 1.0 + contentScale(context) * MediaQuery.textScaleFactorOf(context),
```

Comment:
Sorry for not catching this earlier but - the way the `textScaleFactorOf()` is being applied on this line it only multiplies the second term.  There should be parens around both terms as in the fix below.

Reply:
My bad, you are correct